update-motd.d/99-esm

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+ESM_ENABLED="This Ubuntu 12.04 ESM system is actively receiving extended security updates
```

Comment:
"actively receiving" would be inaccurate, for example if the token has been revoked.